lib/iris/tests/unit/fileformats/pyke_rules/compiled_krb/fc_rules_cf_fc/test_build_verticalp_coordinate_system.py

```diff
@@ -43,6 +45,12 @@ def _test(self, inverse_flattening=False):
             ellipsoid_kwargs['semi_minor_axis'] = 6356256.909
         cf_grid_var_kwargs.update(ellipsoid_kwargs)
 
+        if no_offsets:
+            del cf_grid_var_kwargs['false_easting']
+            del cf_grid_var_kwargs['false_northing']
+            test_easting = 0
+            test_northing = 0
+
```

Comment:
The format of [test_build_albers_equal_area_coordinate_system.py](https://github.com/SciTools/iris/blob/ad1449752892a53a332b760b1658db4a09cb8409/lib/iris/tests/unit/fileformats/pyke_rules/compiled_krb/fc_rules_cf_fc/test_build_albers_equal_area_coordinate_system.py) (and several others) is clearer than this. Should mirror that format here for clarity and also consistency, both of which will aid in future maintenance.

Reply:
@pp-mo and I have agreed that the inconsistencies/problems in the pyke rules testing needn't be addressed given #3415.